### DISCUSSION TITLE
Utility of Pro Meilisearch Integration and syncing.

### DISCUSSION THREAD

**Discussion by davhad** [AntlerVC/firetable#1309] [Support / Q&A] [answered]
Hello,

After going pro and setting up meilisearch the rowy dashboard searching of table records works fine. 

I activated search so that I can add a reference in a table A record with a record in table B by searching for it and selecting from a dropdown.

How can that be done?

Also, how does sync mean in the Rowy search settings page and why is it then when i turn sync on it goes back to unsynced?

Thank you.

**Comment by BeeBombshell** [comment]
Hey @davhad, to add a reference from Table B to a record in Table A you could utilize the Connector Field. You should check out this example to refer to a Firestore Reference: https://docs.rowy.io/field-types/connector#connect-to-a-firestore-collection.

> Also, how does sync mean in the Rowy search settings page and why is it then when i turn sync on it goes back to unsynced?

Sync basically means syncing your documents to the Meilisearch Index such that they are searchable. Not sure about this issue... Could you share a recording via jam.dev so that we can take a look?

**Reply by davhad** [reply]
Thank you for your response: Here's your jam.dev video regarding the second issue: https://jam.dev/c/2300cacf-1a40-41de-a9da-efb0be494df9

**Reply by davhad** [reply]
For the first part: 

1. what would be the difference between a connector field and a connect table field? 

2. In the connector field example you've shown it makes no reference to meilisearch which confuses me as to why meilisearch is needed if we can't without any complicated configuration search a record in table B whose reference should be added to table A. That should be a plain vanilla example specially that in Airtable that's a simple record relation field.

3. I see meilisearch is used in some places (table search) and algolia in others (connector table alpha). Doesn't that mean double the cloud function syncing load on each update?

**Reply by BeeBombshell** [reply, marked as answer]
> 1. what would be the difference between a connector field and a connect table field?

@davhad, regarding your question about the difference between a connector field and a connect table field, the Connect Table is actually an older version of the Connector field type and it has now been **deprecated**. The Connector field is the recommended approach for creating relationships between tables in Rowy.

> 2. In the connector field example you've shown it makes no reference to meilisearch which confuses me as to why meilisearch is needed if we can't without any complicated configuration search a record in table B whose reference should be added to table A. That should be a plain vanilla example specially that in Airtable that's a simple record relation field.

As for the mention of Meilisearch and Algolia, Meilisearch is specifically required for the Full Text Search feature in Rowy. This feature allows you to search all the documents within your table. If you need to set up Full Text Search, you can refer to the documentation at https://docs.rowy.io/full-text-search for detailed instructions.

The Connector field, on the other hand, is a very versatile field type. It allows you to execute backend code and retrieve a list of options to present to the user. These options can be a static list, a list of documents from a collection, or even a list of documents obtained from a query to a third-party API. This flexibility makes the Connector field a powerful tool for customizing the options available in your tables.

> 3. I see meilisearch is used in some places (table search) and algolia in others (connector table alpha). Doesn't that mean double the cloud function syncing load on each update?

As mentioned above, Meilisearch is only required for the Full Text Search feature and isn't required for the Connector field. Also, the Connect Table field which uses Algolia is now deprecated. Depending upon your requirement, feel free to remove either or both of these integrations to avoid any excessive load.

Hope this helps!
